examples/README.md

```diff
@@ -0,0 +1,31 @@
+# Machine Learning Workflows in T3C
+
+## Quickstart
+
+To run the T3C pipeline with a local front-end & server:
+1. Client-side: update `next-client/.env` to point to the report generation endpoint (e.g. `export PIPELINE_EXPRESS_URL=http://localhost:8080/generate`) and run `npm run dev`
+2. Server-side: update `express-pipeline/.env` with OpenAI/Anthropic/GCS keys and run `npm i && npm run dev`
+3. Navigate to `localhost:3000` and upload a CSV file from the UI. Make sure the CSV is formatted as follows 
+
+## Expected CSV format
+
+T3C expects a CSV file in a specific format:
+* there is ***no newline*** at the end of the file (which Pandas adds by default with `dataframe.to_csv()`)
```

Comment:
if I'm understanding correctly, and the pipeline breaks if the CSV has a newline at the end, can you also add a TODO to have the pipeline strip out the newline automatically? not urgent just good to keep track of the usability improvements we'd want to make